src/bento/components/amp-jwplayer/1.0/base-element.js

```diff
@@ -3,12 +3,12 @@ import {dict} from '#core/types/object';
 
 import {BentoJwplayer} from './component';
```

Comment:
To make my question above more concrete, if we decide to move only the WC/Preact/React layers, then I'd expect the `amp-jwplayer.js` file to stay where it is today

Reply:
If I'm understanding @samouri correctly, I agree with this sentiment, as it fits more in line with "AMP is a downstream user of Bento"